client/src/GlobalStyles.js

```diff
@@ -51,11 +52,14 @@ const GlobalStyles = createGlobalStyle`
       @font-face {
          font-family: "OxygenBold";
          src: url(${OxygenBold}) format("truetype");
-         
          font-family:"OxygenBold";
        }  
-  
 }
+         h1 {letter-spacing: 0.2rem;
```

Comment:
strange formatting, maybe prettier fails on this file? 🤷‍♂️

Reply:
Yes, sadly prettier doesn't work on globalStyles - do you have a solution for this?